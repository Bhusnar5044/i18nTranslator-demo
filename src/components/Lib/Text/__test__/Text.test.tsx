import { render } from '@test-utils';
import { Text } from '../Text';

describe('Text', () => {
    it('should render properly', () => {
        const { container } = render(<Text />);
        expect(container).toMatchSnapshot();
    });
});
