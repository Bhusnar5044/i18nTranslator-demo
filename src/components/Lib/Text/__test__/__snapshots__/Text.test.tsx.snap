// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text > should render properly 1`] = `
<div>
  <span
    class="sc-eLqIir BTxYD"
    textalign="initial"
    textcolor="inherit"
  />
</div>
`;

exports[`Text should render properly 1`] = `
<div>
  <span
    class="sc-eLqIir BTxYD"
    textalign="initial"
    textcolor="inherit"
  />
</div>
`;
